client/components/tinymce/plugins/wpcom-track-paste/plugin.js

```diff
@@ -46,39 +42,38 @@ function pasteTrack( editor ) {
 		return SOURCE_UNKNOWN;
 	}
 
-	function flattenArray( orgTypes ) {
-		// As per https://html.spec.whatwg.org/multipage/interaction.html#datatransfer
-		// types should be an array containing the available formats that were set on copy.
-		// Some browsers, as Firefox, return a DOMStringList instead, see
-		// https://developer.mozilla.org/en-US/docs/Web/API/DOMStringList
-		let types = [];
-		if ( orgTypes.contains ) {
-			types = Array.from( orgTypes );
-		} else {
-			types = orgTypes.slice();
-		}
-		return types.reduce( ( a, b ) => {
-			return a + ', ' + b;
-		} );
+	function toString( types ) {
+		// types should be an array. Some browsers, as Firefox, return a DOMStringList instead.
+		// We need coverage for thoses cases, hence the Array.from conversion.
+		//
+		// https://html.spec.whatwg.org/multipage/interaction.html#datatransfer
+		// DOMStringListhttps://developer.mozilla.org/en-US/docs/Web/API/DOMStringList
```

Comment:
Moved as a JSDoc comment. I don't have a strong opinion about the `toString` name, but I think it is OK to have it as it is, as it conveys a well know meaning.